Gemfile

```diff
@@ -0,0 +1,15 @@
+source "https://rubygems.org"
```

Comment:
`Gemfile` allows Ruby developers to clone your repo and then run `bundle install` to get all the gems they need to run your code.